core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/download-requirements.sh

```diff
@@ -597,6 +589,8 @@ get_requirements_from_group 'IMAGES'               'images'                "$REQ
 
 # === Packages ===
 
+check_connection yum $(yum repolist --quiet | tail -n +2 | cut -d' ' -f1 | cut -d'/' -f1 | sed 's/!//')
```

Comment:
```suggestion
check_connection yum $(yum repolist --quiet | tail -n +2 | cut -d' ' -f1 | cut -d'/' -f1 | sed 's/^!//')
```
yum repolist output's first column as ! if the repository has expired metadata.